outputs.tf

```diff
@@ -624,3 +624,7 @@ output "azs" {
   value       = var.azs
 }
 
+output "vpc_name" {
+  description = "The name of the VPC"
+  value       = var.name
```

Comment:
I don't think there is `name` as an attribute here - https://www.terraform.io/docs/providers/aws/r/vpc.html

Reply:
Correct, it's not an exported reference from the aws provider, it's something that's being pulled from the input variables (like AZS above it there). Our organization has a small number of VPCs so we've found it helpful to reference the name from a remote_state lookup.

Reply:
Right, let's call output `name` instead of `vpc_name`. Make a change and I will merge it right away.

Reply:
Ahh yes, you're correct, that's the better method. I'll make that change.